### DISCUSSION TITLE
Does Next.js 13 AppDir Metadata "keywords" without space between keywords right for SEO?

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#47944] [Help] [answered]
### Summary

I'm using Next.js 13 Appdir and using Metadata to automatically generate <meta> tags for the website. However with "keywords" the <meta> tag is automatically generated like this,

`<meta name="keywords" content="Next.js,React,JavaScript">`

instead of like this (with space):

`<meta name="keywords" content="Next.js, React, JavaScript">`

Does this affect SEO? and is this a bug of the current Next.js Metadata?.

Beta Documents of Next.js here: https://beta.nextjs.org/docs/api-reference/metadata#metadata-object

**Comment by icyJoseph** [comment, marked as answer]
I did some, very quick, search regarding this matter, and many outright deem the keywords meta tag useless. 

- https://developers.google.com/search/blog/2009/09/google-does-not-use-keywords-meta-tag

Of course not everyone's searching on Google, but it does look like various Search Engines don't use it at all. So if you have a Search Engine you care about, you better find out if they parse the keywords meta, and what kind of structure they favour.

- https://blogs.bing.com/webmaster/2014/10/03/blame-the-meta-keyword-tag/

As for the spaces, it's just more bytes I guess (`, ` shipping two characters where one would suffice) , as there's no actual spec on the shape of the keywords tag content. 

I found this 2005 post where, they commenter says spaces over commas is better, rather than both. It goes to show how the lack of a spec pretty much puts the ball on the Search Engine itself and it is only validated through personal anecdotes.

- https://www.webmasterworld.com/html/9921.htm (Old, perhaps useless info at this point)

**Reply by ghost** [reply]
Thank you so much, I got it 🎯
